gamification/data/labels.py

```diff
@@ -0,0 +1,22 @@
+def used_labels():
+    labels = [
```

Comment:
declare this as a module level CONSTANT, using tuple syntax.

There is no benefit putting it inside a function, because they are not complex objects from the model.